src/app/auth.tsx

```diff
@@ -35,13 +35,12 @@ export const AuthRouting: React.FC<{ children: React.ReactNode }> = ({ children
             router.push(redirect);
 
             // if Client Side Routing is not working, fallback to changing the window location
-            // This seemingly only happens on GH Actions
             setTimeout(() => {
                 const redirect = toRedirectTo();
                 if (redirect) {
                     window.location.pathname = redirect;
                 }
-            }, 500);
+            }, 2500);
```

Comment:
to confirm, is this 'after 2.5s, if not already redirected, redirect manually'?
why does this cause a white flash?

Reply:
Yep that's correct. When not using client side routing, Next does the same thing that it does on an initial page load and sends an SSRed page, which then gets styled and picks up themeing in local storage etc. all on hydration